custom_components/netro_watering/translations/es.json

```diff
@@ -0,0 +1,130 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "Dispositivo ya configurado"
+        },
+        "error": {
+            "invalid_serial_number": "Número de serie incorrecto",
+            "unknown": "Error inesperado",
+            "unknown_device_type": "Tipo de dispositivo desconocido"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "device_name": "Nombre del dispositivo",
+                    "device_type": "Tipo del dispositivo",
+                    "serial_number": "Número de serie del dispositivo"
+                },
+                "description": "Por favor, introduzca el número de serie de su dispositivo",
+                "title": "Configure Netro"
+            }
+        }
+    },
+    "entity": {
+        "binary_sensor": {
+            "iswatering": {
+                "name": "¿Está regando ahora?",
+                "state": {
+                    "off": "No",
+                    "on": "Sí"
+                }
+            }
+        },
+        "sensor": {
+            "battery_percent": {
+                "name": "Nivel de batería"
+            },
+            "humidity": {
+                "name": "Nivel de humedad"
+            },
+            "illuminance": {
+                "name": "Luz del sol"
+            },
+            "last_watering_end_datetime": {
+                "name": "Hora de finalización del último riego"
+            },
+            "last_watering_source": {
+                "name": "Origen del último riego",
+                "state": {
+                    "fix": "Programa",
+                    "manual": "Manual",
+                    "none": "Desconocido",
+                    "smart": "Inteligente"
+                }
+            },
+            "last_watering_start_datetime": {
+                "name": "Hora de inicio del último riego"
+            },
+            "last_watering_status": {
+                "name": "Estado del último riego",
+                "state": {
+                    "executed": "Ejecutado",
+                    "executing": "Ejecutándose",
+                    "none": "Desconocido",
+                    "valid": "Correcto"
```

Comment:
In the Netro Public API it is said : VALID: New schedules, not executed
So, it does not mean correct in my opinion but rather "planned". This is the reason why I translated by "Planifié" in french. But may be you understand it differently !? In any case "valid" does not seem to me a good wording